File: CRM/src/app/shared/eHospital_DongNai_A_NSTL/NS_DM_LoaiTruc_NgoaiGio.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DM_LoaiTruc_NgoaiGio } from './NS_DM_LoaiTruc_NgoaiGio.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DM_LoaiTruc_NgoaiGioService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'LoaiTruc_Id', 'MaLoaiTruc', 'TenLoaiTruc', 'GhiChu', 'TamNgung'];

    List: NS_DM_LoaiTruc_NgoaiGio[] | undefined;
    ListFilter: NS_DM_LoaiTruc_NgoaiGio[] | undefined;
    FormData!: NS_DM_LoaiTruc_NgoaiGio;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DM_LoaiTruc_NgoaiGio";
    }
}

File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DM_LoaiTruc_NgoaiGioService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DM_LoaiTruc_NgoaiGioService : BaseService<NS_DM_LoaiTruc_NgoaiGio, INS_DM_LoaiTruc_NgoaiGioRepository>
    , INS_DM_LoaiTruc_NgoaiGioService
    {
    private readonly INS_DM_LoaiTruc_NgoaiGioRepository _NS_DM_LoaiTruc_NgoaiGioRepository;
    public NS_DM_LoaiTruc_NgoaiGioService(INS_DM_LoaiTruc_NgoaiGioRepository NS_DM_LoaiTruc_NgoaiGioRepository) : base(NS_DM_LoaiTruc_NgoaiGioRepository)
    {
    _NS_DM_LoaiTruc_NgoaiGioRepository = NS_DM_LoaiTruc_NgoaiGioRepository;
    }
    }
    }

